RULES.md

```diff
@@ -317,90 +319,133 @@ Valid submissions must rely on new algorithmic or mathematical ideas and should
 
 ### Tuning
 
-Tuning will be substantially different for the [External](#external-tuning-ruleset) and the [Self-tuning Ruleset](#self-tuning-ruleset) and the individual specifications for each will be described in the following.
+Tuning will be substantially different for the [external](#external-tuning-ruleset) and the [self-tuning ruleset](#self-tuning-ruleset) and the individual specifications for each will be described in the following.
 
-#### External Tuning Ruleset
+#### External tuning ruleset
 
-For each workload, we will run *S\*O* (e.g. *S=5*, *O=20*) hyperparameter settings. All hyperparameter settings will be obtained from the submission-provided workload-agnostic search space with [(quasi)random search](https://arxiv.org/abs/1706.03200). The **trials** will be randomly partitioned into *S* groups of *O* trials each. In each group of the *S* **studies**, the best training times over all *O* settings will be taken into account and the median of the *S* per-study training times will be the final training time (see [Scoring submissions](#scoring) Section). Runs that do not reach the target error of the evaluation metric have an infinite time.
+For each workload, the hyperparameters are tuned using $O=20$ tuning **trials**. To estimate the variance of the results, this tuning will be repeated for $S=5$ **studies**, for a total of $S\cdot O = 100$ different hyperparameter settings. The submitters will provide a workload-agnostic search space and the working group will then return $100$ hyperparameters settings obtained using [(quasi)random search](https://arxiv.org/abs/1706.03200). The working group will also randomly partition these $100$ trials into $5$ studies of $20$ trials each.
 
-- Suggestion: *S=5*, *O=20*.
-- The number of trials that will be performed is known to the submissions.
-- Run on 20 machines in parallel, report the minimum time per study.
-- Submissions should work with a simple random search, within the provided search space.
-- To estimate study variance and to rule out lucky studies, *S* studies will be run.
-- Submissions are always free to perform additional self-tuning while being timed.
+In each study, the fastest training time across the $O=20$ settings will be taken into account and the median of these $5$ per-study training times will be the final training time for the submission on this workload (see [Scoring submissions](#scoring) section). Runs that do not reach the target performance of the evaluation metric have an infinite time. Submissions are always free to perform additional self-tuning while being timed.
```

Comment:
We discussed this in the meeting on June 23rd (see Minutes: https://docs.google.com/document/d/1eMSebuRvSp4vVdb26lJgEv6Lj7mMz5-KOdBjgYfRpck/edit#heading=h.6nd8neqte1l9)

I am happy to quickly discuss this again. I can put it on the agenda for next week. Could you perhaps in this meeting explain your alternative proposal? If I understand you correctly, you would essentially produce 5 performance profile curves for a submission, right? But you would only use the min across all 5 studies for the competition score?

Reply:
In the current rules, the performance profile for a submission is based on its performance on W workloads. (W=8 if we only consider public workloads)
The score on each workload is relative to the best performing submission on that workload, i.e., $\min_{s} t_{w,s}$ where $t_{w,s}$ is the time it takes submission $s$ to reach the targets on workload $w$.

Currently we get $t_{w,s}$ for each submission by taking the median over 5 seeds. Let's call $t_{w,s,i}$ the time to reach targets of submission $s$ on workload $w$ for the $i$ th seed.

Another way we could capture the "performance distribution" of a submission would be to use all of the individual times. So instead of first computing medians and then using W numbers to compute the performance profile of a submission, we would directly use all 5W numbers to compute the profile. The best time for the workload (reference for computing the performance profile) would need to be $\min_s \min_i t_{w,s,i}$ so that all curves still have support $[1, \infty)$.

Why even consider this? It relates to the question you asked at that June 23 meeting. Suppose that a submission gets fast times on 3 out of their 5 seeds, and the other two are very slow. Currently they would only be scored based on the fast time. In the more complex alternative setup, their curve would be impacted by the two slow runs. Similarly, if a submission is fast 2 out of 5 seeds, and the other three are slow, the submission's score in the current rules will be the slower one.

Using all 5 times would reward approaches that have low variance/variability and penalize approaches that have higher variance (but sometimes do exceptionally well).